classes/ActionScheduler_Abstract_QueueCleaner.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/**
+ * Abstract class with common Queue Cleaner functionality.
+ */
+abstract class ActionScheduler_Abstract_QueueCleaner {
+
+	/** @var ActionScheduler_Store */
+	protected $store;
+
+	/**
+	 * Process an individual action.
+	 *
+	 * @param int $action_id The action ID to process.
+	 */
+	protected function process_action( $action_id ) {
```

Comment:
This function was `public` when it was in `ActionScheduler_QueueRunner`. Changing it to `protected` here will break any 3rd party code calling it directly.